pkg/action/cl001/ac006/executor.go

```diff
@@ -77,15 +77,22 @@ func (e *Executor) execute(ctx context.Context) error {
 	var currentMasterPodsHostNetwork int
 	var masterPods []string
 	for _, pod := range masterPodList.Items {
+		// this cronjob pod currently counts against the pods with hostnetwork on master nodes
+		// it will be dropped with the fix for aws-cni v1.7.0, see issue https://github.com/giantswarm/giantswarm/issues/11077
+		if strings.Contains(pod.Name, "aws-cni-restarter") {
```

Comment:
I saw there might be problems when running tests and the aws-cni-restarter is already allocated on a master node. It shortly runs but it's counted against host network pods. As I know we want to drop this with the next release of aws-cni I'd like to ignore it for now.